### DISCUSSION TITLE
Task completion date/time

### DISCUSSION THREAD

**Discussion by naveenroyv** [jotaen/xit#4] [File Format]
The date/time could be logged when a task is marked complete - this can be useful for logging/reporting purposes. Not sure how this would make its way into the syntax though.

**Comment by jotaen** [comment]
I wonder, however, whether this should be part of the file format specification, or whether this would just be up to a tool.

So, e.g., your editor plugin (or whatever you use) could just add a “completion” timestamp to the description text, like:

```
[ ] Do this.
```
and then:
```
[x] Do this. #done=2022-04-01
```

So instead of there being a pre-defined syntax for this, it’s just an additional convention of the tool that you use.

I think in the end this question would depend a bit on how widely such a feature would be used, and whether or not that information needs to be processed programmatically. This discussion is also related: https://github.com/jotaen/xit/discussions/3

**Comment by majkinetor** [comment]
Date format could support full ISO8061 with time:

```
[x] Do this #done=2022-04-01T14:30

[x] Have best second in life -> 2022-04-01T14:30:22
